better documentation for producing notices (#688)
nasa-gcn/gcn.nasa.gov: app/routes/docs/producers.mdx

      ```python
      from gcn_kafka import Producer

>       import json
>       # Connect as a producer.

Reviewer: ```suggestion
      import json

      # Connect as a producer.
```

      # Warning: don't share the client secret with others.
      producer = Producer(client_id='fill me in', client_secret='fill me in')

>       # any topic starting with 'mission.'

Reviewer: ```suggestion

      # any topic starting with 'mission.'
```

      topic = 'mission.example'

>       #Construct a JSON structured notice that follows the GCN Universal Schema, and references your notice's specific schema.

Reviewer: ```suggestion
      # Construct a JSON structured notice that follows the GCN Unified Schema, and references your notice's specific schema.
```

>       example_notice =
>       {'$schema': 'https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json',

Reviewer: This will cause a syntax error. To fix, put the opening brace on the same line as the assignment.

```suggestion
      example_notice = {
          '$schema': 'https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json',
```

      'alert_reporter': {'mission': 'Swift',
         'instrument': 'BAT-GUANO',
         'record_number': 1},
      'alert_info': {'alert_datetime': '2023-01-01T06:08:19.00',
         'alert_tense': 'current',
         'alert_type': 'initial'},
      'datetime': {'trigger_time': '2022-12-31T21:46:05.10'},
      'statistics': {'rate_snr': 14,
         'rate_duration': 0.256,
         'rate_energy_range': {'energy_low': 15, 'energy_high': 350},
         'classification': {'GRB': 1}},
      'identifiers': {'data_archive_page': 'https://swift.gsfc.nasa.gov/archive/700000031',
         'instrument_id': '694216001',
         'instrument_id_alternate': '694215970'}}

>       notice_bytestr = json.dumps(example_notice)
> 
>       #pass the topic and bytestring, and you've published a notice!
>       producer.produce(topic, notice_bytestr)

Reviewer: `json.dumps` returns a string, not a byte string.

```suggestion
      data = json.dumps(example_notice)

      # Pass the topic and data string, and you've published a notice!
      producer.produce(topic, data)
```
